tests/fv3_conf/gfs_v16_run.IN

```diff
@@ -19,6 +19,14 @@ cp    @[INPUTDATA_ROOT]/FV3_input_data/*_table .
 cp    @[INPUTDATA_ROOT]/FV3_input_data/diag_table_gfdlmp diag_table
 cp    @[INPUTDATA_ROOT]/FV3_input_data_gsd/field_table_suite2 field_table
 cp    @[INPUTDATA_ROOT]/FV3_input_data/*configure .
+for n in 01 02 03 04 05 06 07 08 09 10 11 12; do
+ln -sf @[INPUTDATA_ROOT]/FV3_input_data_INCCN_aeroclim/MERRA2/merra2.aerclim.2003-2014.m${n}.nc aeroclim.m${n}.nc
```

Comment:
Shouldn't we use `cp` here instead of `ln`? Links will break the portability of the rt directory, I can't just copy it to some other machine.

Reply:
isn't this just for the actual run directory, linking the files from the input directory to the run directory?

Reply:
I think tar, cp and scp all have an option to follow symbolic links which can be used when moving run directory to different location/machine.